ci-operator/jobs/openshift/release/openshift-release-release-4.8-periodics.yaml

```diff
@@ -1,8 +1,8 @@
 periodics:
 - agent: kubernetes
   cluster: build03
+  cron: 41 3 */14 * *
```

Comment:
We can fix these up in any way you see fit. In this PR or after the fact, but `promote-...machine-os-content..` jobs need to continue to run often. They are more of an infra job than a test.

Reply:
ill address this in the script to skip `promote-*`

Reply:
Ack, I think we are close finishing this task, I'll leave promotion jobs to you @bear-redhat